bin/oref0-setup.sh

```diff
@@ -194,6 +214,10 @@ if openaps vendor add --path . mmeowlink.vendors.mmeowlink 2>&1 | grep "No modul
         echo -n "Cloning mmeowlink dev: "
         (cd ~/src && git clone -b dev git://github.com/oskarpearson/mmeowlink.git) || die "Couldn't clone mmeowlink dev"
     fi
+	if [[ "$radio_locale" -eq "WW" ]]; then
+		echo -n "Patching mmtune.py for WW frequencies. " # temporary fix for https://github.com/oskarpearson/mmeowlink/issues/16
+		sed -i "s/def __init__(self, link, pumpserial, radio_locale='US'):/def __init__(self, link, pumpserial, radio_locale='WW'):/g" ~/src/mmeowlink/mmeowlink/mmtune.py || die "Couldn't patch mmtune.py"
```

Comment:
Would be nice to make this part of pump.ini so we don't have to hard-code it. In the mean time, though, if you want to do this we need to add a git commit of the change so it doesn't block pulling future updates to that file.